internal/internal_workflow_testsuite.go

```diff
@@ -2279,6 +2279,14 @@ func (env *testWorkflowEnvironmentImpl) executeChildWorkflowWithDelay(delayStart
 	go childEnv.executeWorkflowInternal(delayStart, params.WorkflowType.Name, params.Input)
 }
 
+func (wc *testWorkflowEnvironmentImpl) ExecuteNexusOperation(params executeNexusOperationParams, callback func(*commonpb.Payload, error), startedHandler func(opID string, e error)) int64 {
+	panic("TODO")
```

Comment:
Do we have an issue for this?

Reply:
I just put it here in the PR checklist.
I started working on. I think I'll be able to submit it today though, it's not too bad.
I had to make a test env `Client` with support for some basic methods because the operations all require one.

Reply:
https://github.com/temporalio/sdk-go/pull/1475